### PR TITLE
HOTFIX: pyqt6 version and ginga connection timeout

### PR DESCRIPTION
**Pull request by kbwestfall**
As titled.  Note this is going straight to `release`.

### CODE REVIEW

**Review by jhennawi — APPROVED** [review]
Thanks!

**Comment by jhennawi** [comment]
Will you also pull this into develop when approved?

**Comment by kbwestfall** [comment]
> Will you also pull this into develop when approved?

Yes, but it will also require a PR.

File: setup.cfg [hunk 1]
```diff
@@ -33,7 +33,7 @@ python_requires = >=3.10,<3.13
 setup_requires = setuptools_scm
 include_package_data = True
 install_requires =
-    numpy>=1.23
+    numpy<2.0.0
```
Comment (debora-pe) [inline setup.cfg]: Quick question. If before this needed to be `>=1.23`, should it now still need to be `>=1.23` and `<2.0.0`?

**Review by debora-pe — APPROVED** [review]
One quick question

File: setup.cfg (hunk 1 above)
Comment (kbwestfall) [reply setup.cfg]: Yes!  Good catch!  Fixed.

**Pull request merged**